packages/remix-dev/compiler/plugins/serverBareModulesPlugin.ts

```diff
@@ -22,7 +22,9 @@ export function serverBareModulesPlugin(
   remixConfig: RemixConfig,
   onWarning?: (warning: string, key: string) => void
 ): Plugin {
-  let isDenoRuntime = remixConfig.serverBuildTarget === "deno";
+  let isDenoRuntime = ["deno", "netlify-edge"].includes(
```

Comment:
Deno essentially forks typescript and has their own module resolution algorithm. Using the standard tsconfig won't suffice as it doesn't account for URL imports. When building for deno, we add an esbuild plugin that uses the deno resolution algorithm.

Reply:
Gee, thanks Deno! In that case it sounds like maybe we need another top-level config option (since `serverBuildTarget` is going away) that controls how we resolve modules... maybe something like:

```
moduleResolution: "node" | "tsconfig" | "deno"
```

Does that sound reasonable?